src/pages/ProductDetail.tsx

```diff
@@ -0,0 +1,24 @@
+import { useParams } from 'react-router-dom'
+import useProductDetail from '../hooks/useProductDetail'
+
+const ProductDetail = () => {
+  const { id } = useParams()
+  const productDetail = useProductDetail(id!, '../../products.json')
```

Comment:
eslint warning, use a default value by pass
![image](https://user-images.githubusercontent.com/28136235/229850237-0c3c9f60-f44d-448a-937f-bf9ad5767dc4.png)
```suggestion
  const { id = ''} = useParams()
  const productDetail = useProductDetail(id, '../../products.json')
```

Reply:
Applied suggestion.